tools/distrib/python/grpcio_tools/grpc_tools/_protoc_compiler.pyx

```diff
@@ -40,13 +43,15 @@ cdef extern from "grpc_tools/main.h" namespace "grpc_tools":
                         vector[pair[string, string]]* files_out,
                         vector[cProtocError]* errors,
                         vector[cProtocWarning]* wrnings) nogil except +
-  int protoc_get_services(char* protobuf_path,
+  int protoc_get_services(char* protobuf_path, char* version,
                           vector[string]* include_path,
                           vector[pair[string, string]]* files_out,
                           vector[cProtocError]* errors,
                           vector[cProtocWarning]* wrnings) nogil except +
 
 def run_main(list args not None):
+  version = "--grpc_tools_version={}".format(grpc_version.VERSION)
```

Comment:
Instead of using this flag, can you just add a parameter to `protoc_main`? We should be completely in control of the signature.